perception_dataset/rosbag2/rosbag2_to_non_annotated_t4_converter.py

```diff
@@ -174,7 +174,7 @@ def _calc_actual_num_load_frames(self):
         num_frames_to_crop = 0
 
         if not (self._num_load_frames > 0 and self._num_load_frames <= max_num_frames):
```

Comment:
Not sure if this is correct since still catching up with this whole codebase yet, but shouldn't we also modify here as well?

```suggestion
        if not (self._num_load_frames > 0 and self._num_load_frames <= max_num_frames - 1):
```

Reply:
Thanks! I fixed as de266896903ac78a9e0c04d5f7d50ad41ba12352